Test/Integration Test/Readme.md

```diff
@@ -0,0 +1,26 @@
+# How to run cypress test successfully in your environment 
+
+In order to run the test cases successfully, you need to follow the mentioned guidlines.
+
+## Prerequisites: 
+
+1) Magento2 should be installed on publically accessible URL
+2) Cypress should be installed
+
+## Steps 
+
+1) First we need to install required packages using `npm i`
+Next, to set the your URL and Credentials please proceed by following mentioned steps:
+
+2) Search for the file "cypress/fixtures/config.json" and Change the parameters below according to your environment and save the file.
+
+{
+  "url": "http://sampledomain.com",
+  "email": "demo@mail.com",
+  "pass": "password",
+  "adminURL":"http://sampledomain.com/admin",
+  "adminUsername":"admin_user",
+  "adminPass":"admin_password"
+}
+
+3) Run `npm run cypress_run` to run all the tests
```

Comment:
Add this to the main README.md instead in the section 
## Integration Tests
Make sure than you use a developer friendly guide, with less test and more commands, so it is very easy to spot what needs to be done.

## Steps

1) Install dependencies
```bash
npm i
```
2) Update `cypress/fixtures/config.json`
3) Run test 
```bash
npm run cypress_run <- Please provide actual path
```